ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Constants/Constants.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ExadelTimeTrackingSystem.BusinessLogic.Constants
```

Comment:
you don't need Constants folder, can put Constants.cs file just in the root of BusinessLogic